### PR TITLE
feat: make some updates to the rp3 page

### PR DESCRIPTION
**Pull request by avneesh0612**

### CODE REVIEW

File: components/hackathon/common/AvatarShowcase.tsx [hunk 1]
```diff
@@ -20,13 +20,12 @@ export const AvatarShowcase: React.FC<AvatarShowcaseProps> = ({
   return (
     <VStack mb={20} spacing={12}>
       <Heading size="title.2xl">{title}</Heading>
-      <SimpleGrid
```
Comment (nachoiacovino) [inline components/hackathon/common/AvatarShowcase.tsx:23]: why? these makes the columns not the same size
image

**Review by nachoiacovino — CHANGES_REQUESTED** [review]

File: components/hackathon/common/AvatarShowcase.tsx (hunk 1 above)
Comment (avneesh0612) [reply components/hackathon/common/AvatarShowcase.tsx:23]: we won't be able to center it otherwise, right?

Comment (nachoiacovino) [reply components/hackathon/common/AvatarShowcase.tsx:23]: is that super relevant? I very much prefer it to be aligned than centered

**Review by nachoiacovino — APPROVED** [review]
lgtm!

**Pull request merged**